Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -5,20 +5,20 @@
 #include <algorithm>
 #include <cstdio>
 #include <memory>
+#include <utility>
 
 #include "Common/CommonTypes.h"
 #include "Common/FileUtil.h"
 #include "DiscIO/CISOBlob.h"
 
 namespace DiscIO
 {
-static const char CISO_MAGIC[] = "CISO";
-
-CISOFileReader::CISOFileReader(std::FILE* file) : m_file(file)
+CISOFileReader::CISOFileReader(File::IOFile&& file) : m_file(std::move(file))
```

Comment:
But I still have to use `m_file(std::move(file))` instead of `m_file(file)`, right? Wouldn't it be problematic to use `File::IOFile file` with `m_file(std::move(file))` if someone makes `IOFile` copyable in the future?